testsuite/classlibrary/TestServer_clientID_booted.sc

```diff
@@ -70,4 +70,48 @@ TestServer_clientID_booted : UnitTest {
 		s.quit;
 		s.remove;
 	}
+
+	test_recoverRemoteReLogin {
+		var options, homeServer, remote, dt = 0.1, timeout = 5;
+
+		options = ServerOptions.new;
+		options.maxLogins = 4;
+		options.sampleRate = 48000;
+
+		homeServer = Server(\pseudoHome, options: options);
+
+		homeServer.clientID = 3;
+		this.bootServer(homeServer);
+
+		this.assert(homeServer.clientID == 3,
+			"homeServer gets requested clientID back from server process."
+		);
```

Comment:
this should be a separate test or removed altogether

Reply:
ok, done